packages/react-server/core/ReactServerAgent.js

```diff
@@ -10,8 +11,14 @@ function makeRequest (method, url) {
 	return new Request(method, url, API.cache());
 }
 
+const DATA_BUNDLE_CACHE     = {};
```

Comment:
Still trying to read through this PR, but this looks like a module-level variable? Does it have request-specific stuff in it?